internal/transform/contract_events.go

```diff
@@ -0,0 +1,144 @@
+package transform
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/stellar/stellar-etl/internal/toid"
+	"github.com/stellar/stellar-etl/internal/utils"
+
+	"github.com/stellar/go/ingest"
+	"github.com/stellar/go/strkey"
+	"github.com/stellar/go/xdr"
+)
+
+// TransformContractEvent converts a transaction's contract events and diagnostic events from the history archive ingestion system into a form suitable for BigQuery.
+// It is known that contract events are a subset of the diagnostic events XDR definition. We are opting to call all of these events
+// contract events for better clarity to data analytics users.
+func TransformContractEvent(transaction ingest.LedgerTransaction, lhe xdr.LedgerHeaderHistoryEntry) ([]ContractEventOutput, error) {
+	ledgerHeader := lhe.Header
+	outputTransactionHash := utils.HashToHexString(transaction.Result.TransactionHash)
+	outputLedgerSequence := uint32(ledgerHeader.LedgerSeq)
+
+	transactionIndex := uint32(transaction.Index)
+
+	outputTransactionID := toid.New(int32(outputLedgerSequence), int32(transactionIndex), 0).ToInt64()
+
+	outputCloseTime, err := utils.TimePointToUTCTimeStamp(ledgerHeader.ScpValue.CloseTime)
+	if err != nil {
+		return []ContractEventOutput{}, fmt.Errorf("for ledger %d; transaction %d (transaction id=%d): %v", outputLedgerSequence, transactionIndex, outputTransactionID, err)
+	}
+
+	// GetDiagnosticEvents will return all contract events and diagnostic events emitted
+	contractEvents, err := transaction.GetDiagnosticEvents()
```

Comment:
The go monorepo converts all contract_events to diagnostic_events.

I want to rename them all back to contract_events as in diagnostic_events are now call contract_events as well because that makes more sense to me from a user perspective. Like I'd rather have `history_contract_events` instead of `history_diagnostic_events`